README.md

```diff
@@ -752,6 +752,63 @@ Coherence is a user management and authentication solution. Support for authoriz
 
 For an example of using [Canary](https://github.com/cpjk/canary) with Coherence, please visit the [CoherenceDemo canary branch](https://github.com/smpallen99/coherence_demo/tree/canary).
 
+## Contexts
+
+Phoenix 1.3 promotes contexts as an abstraction on top of Ecto schemas or models.
```

Comment:
I don't think we need to explain Phoenix contexts here. Simply explain that a new context can be generated for the Coherence user schema.